README.md

```diff
@@ -1,8 +1,48 @@
+
+**Python Data Analytics** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_awaiszafar/tree/masterhttps://github.com/jarviscanada/jarvis_data_eng_AwaisZafar/tree/feature/python-data-analytics)]: A Syetem Required Document was prepare containing data models using E-R Diagrams, and other project objectives. A proof of concept(POC) project to understand customer shopping behavior for the  London Gift shop(LGS) which is an online giftware store in the UK that has been running for more than 10 years. The revenue has not grown in recent years so the LGS marketing team wants to utilize the latest data technologies to understand the customer better. LGS team hires Jarvis data engineering services since the LGS IT dept doesn't have enough resources to work on more projects. The LGS marketing team will use the final data analytics product to develop targeted marketing campaigns to attract new or existing customers. The project was divided into two parts. First is setting up a data warehouse for Online analytical processing (OLAP) operations which were done by dumping LGS retail data into the PostgreSQL database which was provisioned using a docker container. The second part of the project was to develop a notebook that answers business questions compiled by the data analysts. The technologies that were used to solve the business questions were Python, Jupyter Notebook which was also provisioned using a docker container, Pandas Dataframe, and Numpy. The final product was deployed using Github.
+
+
+## Highlighted Projects
+**Personal Portfolio Webpage** [[GitHub](https://codepen.io/awais-zafar/full/VwrmjOe)]: Designed a webpage using HTML5 and CSS as the first step toward getting into the software industry which included the links to all other similar front-end projects as part of my self-study front-end development course. This project aimed to use all the features provided by the tools like HTML5 and CSS to design a webpage.
+
+**Particle Flow Monitor**: A business required document (BRD) was presentated and I was required to prepare System Required Document (SRD) containing flowcharts of the software design. Developed an online continuous particle flow monitor in pneumatic conveyed pipelines using electrostatic sensors and Arduino as microcontroller and C-code was written to implement the graphical user interface which returns the values measured by the electrostatic sensors and stored into excel database and extracted if need be through Matlab for graphical representation. As part of my bachelor's final year project including 100 pages thesis was prepared to document all the details of the project and eventually, the project was presentated using .ppt upon completion. As it was an individual project so, a lot of research was done and implemented. It helped me to understand the different factors which need to be taken into consideration for these devices.
```

Comment:
**Particle Flow Monitor**: A business required document (BRD) was presented, and I was required to prepare a System Required Document (SRD) containing flowcharts of the software design. Developed an online continuous particle flow monitor in pneumatic conveyed pipelines using electrostatic sensors and Arduino as a microcontroller, and C-code was written to implement the graphical user interface, which returns the values measured by the electrostatic sensors and stored into an excel database and extracted if need be through Matlab for graphical representation. As part of my bachelor's final year project, including 100 pages thesis was prepared to document all the details of the project and eventually, the project was presented using .ppt upon completion. As it was an individual project so, a lot of research was done and implemented. It helped me to understand the different factors which need to be taken into consideration for these devices.